Sun_Shengran/progress.md

```diff
@@ -3,78 +3,71 @@
 **Project: Recycle Bin**
 
 Overall Direction:
-- 
-- 
-- 
+- Build a recovery page for instructors to display user deleted courses, sessions and students
+- Allow instructors to recover deleted courses, sessions and students from the recycle bin
+- Allow instructors to permanently remove selected/all items from the recycle bin
+- Build UI presentation for the deleted items and relevant functions
+- Auto remove items from the recycle bin after a certain period
+- Create comprehensive tests for the recovery feature
 
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Build UI frame/structure for displaying of deleted courses, sessions and students
+- [ ] Create fake deleted items data for the acceptance testing of UI
```

Comment:
You may not have enough things to do this week. You can start taking a look at the util and database/entity classes to familiarize yourself with the code as well.

Reply:
Okay sure.